flase_app/models.py

```diff
@@ -76,3 +73,13 @@ class CylinderChange(models.Model):
     is_connected = models.BooleanField(blank=True, null=True)
     note = models.CharField(max_length=256, blank=True, null=True)
 
+class SpravaPlynov(models.Model):
```

Comment:
nie uplne rozumiem, co ma tento model reprezentovat

Reply:
pozrieť návrh a skontrolovať či to sedí s týmto modelom